MDANSE/Src/MDANSE/Framework/Converters/MDAnalysis.py

```diff
@@ -229,7 +229,7 @@ def run_step(self, index: int):
 
             if hasattr(self.u.trajectory.ts, "forces"):
                 conf["gradients"] = self.u.trajectory.ts.forces * measure(
-                    1.0, "kJ/mol ang", equivalent=True
+                    1.0, "k_j/mol ang", equivalent=True
```

Comment:
This change causes the MDAnalysis conversion to fail when forces exist.

```
MDANSE.Framework.Units.UnitError: The unit k_j is unknown
```

Reply:
Ah, thought I'd caught all of those, good spot, thanks.